### PR TITLE
Bet model testing

### PR DESCRIPTION
**Pull request by PeterTianjieWen**
Add some validates and a test to bets model. Not working right now. Plz Colton pass.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -74,4 +74,4 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
-gem 'devise'
```
Comment (coltoncorthell) [inline Gemfile]: But why tho

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -42,4 +42,4 @@
     <%= yield %>
   </div>
 </body>
-</html>
```
Comment (coltoncorthell) [inline app/views/layouts/application.html.erb]: Why

File: test/controllers/bets_controller_test.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 require 'test_helper'
 
 class BetsControllerTest < ActionDispatch::IntegrationTest
+
```
Comment (coltoncorthell) [inline test/controllers/bets_controller_test.rb]: WHy

File: test/models/bet_test.rb [hunk 4]
```diff
@@ -14,7 +14,17 @@
...
-  # test "the truth" do
-  #   assert true
-  # end
+
+  
+  def setup
+      @bet = Bet.new(value: 1000, game_id: 1, user_id: 1, team_id: 2)
+  end
+  
+
```
Comment (coltoncorthell) [inline test/models/bet_test.rb]: Remove this spacing but leave one line

**Comment by PeterTianjieWen** [comment]
It's weird cause I didn't move those shits around.

**Pull request merged**